introduction.md

```diff
@@ -0,0 +1 @@
+On my way to becoming a GitHub Master
```

Comment:
How do I reopen this so I can make it to the next step in my course. I was messing around from my Chromebook and am now on a proper laptop. I seem to be stuck at this step.

Thanks!